### DISCUSSION TITLE
Ability to edit profile

### DISCUSSION THREAD

**Discussion by NihalNavath** [community/community#48654] [Mobile]
Users right now are unable to edit profile details like description, change profile picture etc from mobile app. support for this feature would be really useful.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.100.0 (91333308)
Device: iPhone 6S Plus (iOS 15.7.3)
</details>

**Comment by HenSquared** [comment]
Hey @NihalNavath. I'll open a feature request issue and discuss this internally with the team! Thanks for the feedback.

**Reply by NihalNavath** [reply]
Any update on this? :)

**Reply by bjsho** [reply]
any updates?

**Reply by skye-z** [reply]
Looking forward to this update, I need the ability to edit status on mobile :D

**Reply by bjsho** [reply]
> Looking forward to this update, I need the ability to edit status on mobile :D

same

**Reply by 7MinutesDead-Git** [reply]
I'm surprised there's no functionality to do this. I thought maybe my app was bugged or something

I now see that this feature has been requested for at least 2 years. Is it possible to contribute to this in any way?
